src/general/OptionsMenu.cs

```diff
@@ -1114,4 +1114,10 @@ private void UpdateSelectedLanguage(Settings settings)
             languageSelection.Selected = languages.IndexOf(settings.SelectedLanguage.Value);
         }
     }
+
+    private void OnLogButtonPressed()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+        OS.ShellOpen(ProjectSettings.GlobalizePath(Constants.LOGS_FOLDER_NAME));
```

Comment:
Sorry for double review, just tested now. The resulting path is incorrect as the constant only specifies the folder name, so I think this could be changed into:
```suggestion
        OS.ShellOpen(ProjectSettings.GlobalizePath("user://" + Constants.LOGS_FOLDER_NAME));
```

Reply:
No problem 
ok I change that now 
with double slash ? it's not creating a comment after ?
or may be with the quotes ?

Reply:
No the double slash shouldn't comment the line since it's a string (quotation marks).

Reply:
It might actually be better to just change the constant to LOGS_FOLDER and update StartupActions to use that.

Reply:
what do you mean ?Change the log folder path localization in constants file directly ?

Reply:
See the comment in my latest review.